src/ngAria/aria.js

```diff
@@ -83,7 +83,8 @@ function $AriaProvider() {
     ariaMultiline: true,
     ariaValue: true,
     tabindex: true,
-    bindKeypress: true
+    bindKeypress: true,
+    keypressNodeNames: ['DIV', 'LI']
```

Comment:
@caitp revisiting this so we can get a fix in to the next release. I'm still unclear how we would fix the bug with a wildcard for all elements.